<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>交易中心</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/ana.ico</normaloff>../icon/ana.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>手动下单</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCalendarWidget" name="calendarWidget"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>订单</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_23">
              <item>
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>下单时间</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="send_date"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>订单类型</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="order_type">
                <item>
                 <property name="text">
                  <string>期权合约</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>期权标的</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>标的ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="underlying_id"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_20">
              <item>
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>合约ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="contract_id"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>开平标志</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="position_effect">
                <item>
                 <property name="text">
                  <string>开仓</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>平仓</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22">
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>买卖方向</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="side">
                <item>
                 <property name="text">
                  <string>买入</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>卖出</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>收盘价</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="close_price">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>100000000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>委托量</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="mvolume">
                <property name="maximum">
                 <number>100000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="add_order">
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>828</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="mtree">
         <column>
          <property name="text">
           <string>委托下单时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">证券ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>合约ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>开平标志</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>买卖方向</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>订单类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>委托量</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>信号下单</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_25">
       <item>
        <widget class="QTreeWidget" name="backtest_tree">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>设置</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>回测</string>
          </property>
          <item>
           <property name="text">
            <string>期权</string>
           </property>
           <property name="whatsThis">
            <string>option</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>期货</string>
           </property>
           <property name="whatsThis">
            <string>future</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_26">
          <item>
           <widget class="QGroupBox" name="backtest_box">
            <property name="title">
             <string>设置</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>模拟交易</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>开始时间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="start_date">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2016</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_24">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>结束时间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="end_date">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2017</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>运行</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icon/run.png</normaloff>../icon/run.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>账户</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="account"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>开仓类型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="open_type">
             <item>
              <property name="text">
               <string>固定数量</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>固定金额</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>资金比例</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>平仓类型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="close_type">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>全部平仓</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>比例平仓</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QTreeWidget" name="result_tree">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>资产</string>
            </property>
            <property name="whatsThis">
             <string>cash</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>仓位</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>订单</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>策略表现</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>初始资金</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>收益率</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>盈利次数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>亏损次数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>胜算率</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLineEdit" name="init_capital"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="profit_ratio"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="win_count"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="lose_count"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="win_ratio"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>平均盈利</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>平均亏损</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_26">
                  <property name="text">
                   <string>盈亏比</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_27">
                  <property name="text">
                   <string>剩余资金</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>最大回撤</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLineEdit" name="average_win"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="average_loss"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="win_loss"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="final_capital"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="max_draw_down"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="delete_order">
   <property name="text">
    <string>删除订单</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_add_option_underlying">
   <property name="text">
    <string>添加期权标的</string>
   </property>
  </action>
  <action name="action_add_option_group">
   <property name="text">
    <string>添加期权组合</string>
   </property>
  </action>
  <action name="action_add_option_contract">
   <property name="text">
    <string>添加期权合约</string>
   </property>
  </action>
  <action name="action_no_support">
   <property name="text">
    <string>暂时不支持</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
